### ISSUE TITLE
Binaries JAR not downloaded

### ISSUE THREAD

**Issue by kralikba** [alexarchambault/coursier#422]
Hi,

I've added the dependency `"com.siftscience" % "sift-java" % "1.2"` to my project, which has a really simple POM ( https://search.maven.org/remotecontent?filepath=com/siftscience/sift-java/1.2/sift-java-1.2.pom ).
To my greatest surprise, the .jar file is not downloaded by coursier and my project fails to build. (sources, pom, javadoc are there in the cache; furthermore, dependencies are correctly resolved, downloaded, put on classpath.)

Without coursier, SBT handles the task correctly. 

Please give me some pointers on what could be wrong. Thanks!

**Comment by kzys** [comment]
Coursier's CLI can download the dependencies correctly. What was the exact error message you got?

```
% ./cli/target/pack/bin/coursier fetch -t com.siftscience:sift-java:1.2
  Result:
└─ com.siftscience:sift-java:1.2
   ├─ com.google.code.gson:gson:2.7
   ├─ com.squareup.okhttp3:okhttp:3.4.1
   |  └─ com.squareup.okio:okio:1.9.0
   └─ com.squareup.okio:okio:1.9.0
/Users/kazuyoshi/.coursier/cache/v1/https/repo1.maven.org/maven2/com/squareup/okhttp3/okhttp/3.4.1/okhttp-3.4.1.jar
/Users/kazuyoshi/.coursier/cache/v1/https/repo1.maven.org/maven2/com/google/code/gson/gson/2.7/gson-2.7.jar
/Users/kazuyoshi/.coursier/cache/v1/https/repo1.maven.org/maven2/com/squareup/okio/okio/1.9.0/okio-1.9.0.jar
%
```

**Comment by kralikba** [comment]
No error messages, just that `sift-java-1.2.jar` is not downloaded. I'm not familiar with the CLI, but if the output filenames are the newly downloaded ones, you have reproduced the problem.

**Comment by kzys** [comment]
I see. This is, probably because the packaging element the pom has.

```
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.siftscience</groupId>
  <artifactId>sift-java</artifactId>
  <version>1.2</version>
  <packaging>pom</packaging>
```

Since `pom` is specified, Coursier downloads the pom file instead of the jar file.

**Comment by kzys** [comment]
Related #278

**Comment by kralikba** [comment]
Thanks, the workaround mentioned in #278's comments worked.
For reference:
`"com.siftscience" % "sift-java" % "1.2"  artifacts(Artifact("sift-java"))`

**Comment by alexarchambault** [comment]
This should be fixed in the current master, and will be included in the upcoming `1.0.0-M15-2`. Feel free to re-open if you still face this issue.
